README.md

```diff
@@ -4,7 +4,7 @@
   </a>
 </p>
 <h1 align="left">
-   
+
```

Comment:
I'd prefer to have a separate commit/PR for these formats. While it's not a major issue here, It is confusing.

Reply:
my IDE does it automatically

Reply:
@jtakalai Then just commit the formats first